Searching Algorithms/Binary Search Algorithm/C++/binary-search.cpp

```diff
@@ -55,15 +58,15 @@ int main() {
 	int key;
```

Comment:
Now that I look at it again, I feel `key` could be renamed to `val` instead. "Searching for a value" sounds more friendly than "searching for a key". Please rename this variable to `val` here and also in the `binarySearch` function.